src/App.js

```diff
@@ -39,15 +41,19 @@ export default function App() {
     resumeState();
   }, [resumeState]);
 
-  const props = { ...resumedState };
-
-  return !resumedState ? (
-    <></>
-  ) : (
-    <ConfirmContextProvider>
-      <AppContextProvider {...props}>
-        <Layout />
-      </AppContextProvider>
-    </ConfirmContextProvider>
-  );
+  const props = { ...filteredState };
+
+  return !filteredState
+    ? (
+      <></>
```

Comment:
When there is a permalink that needs to load data from DCS, this can take some time and causes the screen to be grayed out for a while if the connection is not fast enough. A loading spinner could be useful in this case, although I prefer the skeleton effect: https://mui.com/material-ui/react-skeleton/, so the app can load certain parts of the UI and the ones that still need data that is still loading could render a skeleton in the meantime.

Reply:
> Would like to schedule a meeting to discuss the DCS-API and how that avoids conflicts with the GiteaReactToolkit that is already implemented.
> 
> We can also discuss how we might start abstracting the DCS-API hooks so that we can use them in other apps.
> 
> Again, Great job!

Sure, I'm on board. When would be a good date and time for you?